src/webgpu/api/operation/command_buffer/programmable/state_tracking.spec.ts

```diff
@@ -10,6 +10,266 @@ TODO: for each programmable pass encoder {compute pass, render pass, render bund
 `;
```

Comment:
Need to clean up this TODO (I think it's sufficiently done now - though maybe there should be a new TODO for trying to do really complex or pseudorandom sequences of things? rockot may have also thought about this in stress tests, I don't remember)